frontend/coreproject/src/routes/+layout.svelte

```diff
@@ -107,11 +107,11 @@
 	}
 </script>
 
-<div class="h-screen overflow-hidden">
+<div class="h-screen overflow-hidden bg-surface-500">
```

Comment:
`bg-surface-500`

Dont add this. Skeleton does this by default

Reply:
No, the background added from skeleton is not same as in the design

Reply:
> No, the background added from skeleton is not same as in the design

What do you mean by this

Reply:
When I remove that class I'm not gettting same color, more darker color